packages/manager/modules/manager-components/src/region/constants.js

```diff
@@ -12,7 +12,7 @@ export const COUNTRIES = {
   VIN: 'US',
   HIL: 'US',
   SGP: 'SG',
-  YYZ: 'CA',
+  TOR: 'TOR',
```

Comment:
Should it not remain `CA` here ?

```suggestion
  TOR: 'CA',
```